src/fetch/payouts.py

```diff
@@ -98,6 +103,9 @@ def from_series(cls, frame: Series) -> RewardAndPenaltyDatum:
             if not math.isnan(frame["eth_slippage_wei"])
             else 0
         )
+        mev_kickback = (
+            int(frame["refund_wei"]) if not math.isnan(frame["refund_wei"]) else 0
```

Comment:
Assuming we want to handle all the missing values the same, we could replace this and the above with:

```python
frame.fillna(0)
```

Alternatively, pandas already has a `.isna` function in it. Skipping the math import.

Reply:
Yes please -- this is what I have needed all my life.